cloud/src/definer/crud.definer.js

```diff
@@ -166,16 +166,16 @@ Parse.Cloud.define('postObjectsToClassWithRelation', (request) => new Promise((r
   let loopedJson = {};
   let newFieldsArray = [];
   Object.keys(localObject).forEach((key) => {
-    const obj = localObject[key];
-    if (!obj.includes('data:image/jpg;base64,')) {
```

Comment:
@jamccomb92 

not all they keys are `Strings` so if you tried to `includes()` on an object or an array, this broke.